www/include/options/accessLists/resourcesACL/DB-Func.php

```diff
@@ -577,9 +586,78 @@ function updateMetaServices($acl_id = null)
     if (isset($ret)) {
         foreach ($ret as $key => $value) {
             if (isset($value)) {
-                $DBRESULT = $pearDB->query("INSERT INTO acl_resources_meta_relations (acl_res_id, meta_id) 
+                $DBRESULT = $pearDB->query("INSERT INTO acl_resources_meta_relations (acl_res_id, meta_id)
                     VALUES ('".$acl_id."', '".$value."')");
             }
         }
     }
 }
+
+/**
+ * sanitize resources parameter for Create / Update a Resource ACL
+ *
+ * @param array<string, mixed> $resources
+ * @return array<string, mixed>
+ */
+function sanitizeResourceParameters(array $resources)
```

Comment:
Argument type hinting is available in PHP 5.x ?

Reply:
yes, but not for every type (array is ok)